configs/config.yaml

```diff
@@ -0,0 +1,12 @@
+defaults:
+    - _self_
+    - env: upgraded_bin_pack  # [bin_pack, cleaner, connector, cvrp, game_2048, graph_coloring, job_shop, knapsack, maze, minesweeper, mmst, multi_cvrp, robot_warehouse, rubiks_cube, snake, sudoku, tetris, tsp]
```

Comment:
This is a mistake, I changed the name of my class yesterday before creating the PR and used the IDE's ability to refactor to make sure it changed everywhere. I guess this hasn't changed since it's a string , I'll change this immediately.